src/collective/cover/tests/cover.robot

```diff
@@ -44,7 +44,8 @@ Setup Cover Test Case
 Click Add Cover
     Open Add New Menu
     Click Link  css=a#collective-cover-content
-    Page Should Contain  Add Cover
+    Wait Until Page Contains  Add Cover
+    Sleep  1s  Wait for js
```

Comment:
`Sleep` should be avoided as it may also fail if the process takes more than 1 second; could you please check if there is another condition that we could check to make sure plone4.csrffixes did its job?